src/main/java/org/springframework/data/elasticsearch/core/convert/GeoConverters.java

```diff
@@ -67,14 +68,14 @@
 	 * {@link Converter} to write a {@link Point} to {@link Map} using {@code lat/long} properties.
 	 */
 	@WritingConverter
-	public enum PointToMapConverter implements Converter<Point, Map<String, Object>> {
+	public enum PointToMapConverter implements Converter<Point, Document> {
```

Comment:
shouldn't we rename all these converters from `MapTo...` and `...ToMap` into `DocumentTo...`and `...ToDocument`?

Reply:
Good point. Let me take care of that.